fullstop/pom.xml

```diff
@@ -68,22 +68,22 @@
         <!-- START PLUGINS -->
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>fullstop-count-events-plugin</artifactId>
+            <artifactId>fullstop-ami-plugin</artifactId>
```

Comment:
why did you move all dependencies? It's pretty difficult to do a review and check what is new and what was moved.